### ISSUE TITLE
investigate label ?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/labels#34]
what about an `investigate` label to indicate that we are looking into an issue...?

**Comment by nelsonic** [comment]
@iteles I don't want to cause _clutter_ by creating the "wrong" labels, but I think _something_ along the lines of `investigate` or `research` is a _useful_ label to have in _most_ projects that require _investigation_ into an issue (_e.g: is the feature required? or can we confirm this is a real "bug"?_)

I could _really_ use this label right now on the https://github.com/dwyl/learn-apple-watch-development project ... :thinking:

**Comment by iteles** [comment]
I've used it on every project so far, though only a handful of times per project.

**Comment by nelsonic** [comment]
yeah, this is very much a _beginning_ of the project label... but I still think it's relevant.
And for a project that keeps iterating/learning it's super useful.
